include/net/net_pkt.h

```diff
@@ -2367,9 +2367,9 @@ int net_pkt_read_be32_new(struct net_pkt *pkt, u32_t *data);
 /**
  * @brief Write data into a net_pkt
  *
- * Note: net_pkt's cursor should be properly initialized and,
- *       eventually, properly positioned using net_pkt_skip_read/write.
- *       Cursor will be updated according to parameters.
+ * Note: net_pkt's cursor should be properly initialized and, if needed,
+ *       positioned using net_pkt_skip_read/write.
```

Comment:
net_pkt_skip/read/write 
there was an error in original doc.

Apply that everywhere